packages/replay/src/metadata/test/v2.ts

```diff
@@ -131,23 +133,23 @@ const v2_1_0 = assign(
   })
 );
 
-const v2_2_0 = assign(
+const v3_0_0 = assign(
   v2_1_0,
   object({
-    tests: array(test_v2_2_0),
+    tests: array(test_v3_0_0),
   })
 );
 
 export namespace TestMetadataV2 {
   export type UserActionEvent = Infer<typeof userActionEvent>;
-  export type Test = Infer<typeof test_v2_2_0>;
+  export type Test = Infer<typeof test_v3_0_0>;
   export type TestResult = Infer<typeof testResult>;
-  export type TestRun = Infer<typeof v2_2_0>;
+  export type TestRun = Infer<typeof v3_0_0>;
   export type TestError = Infer<typeof testError>;
 }
 
 export default {
-  "2.2.0": v2_2_0,
+  "3.0.0": v3_0_0,
```

Comment:
TODO: I discovered that devtools might depend on properties that I wanted to remove. So this version should now be reverted to 2.2.0